Update Move-CsUser.md (#10946)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Move-CsUser.md

## SYNOPSIS

Moves one or more user accounts enabled for Skype for Business Server to TeamsOnly (or the reverse). This cmdlet also can be used to move on-premises users from one pool to another. 

**PRE-REQUISITES**

> - Install or upgrade MicrosoftTeams powershell module with version >= 5.2.0

Reviewer: @ruchiagrawal9221 , This should explicitly state that the below steps are needed only for China environment.

I also don't think we should call out china cloud as Gallatin.
https://learn.microsoft.com/en-us/microsoft-365/admin/services-in-china/services-in-china?view=o365-21vianet
So, need to find the right wordings and add in here.

> - Run Set-TeamsEnvironmentConfig -TeamsEnvironmentName TeamsChina

Reviewer: It is also better to provide link for set-teamsEnvironmentConfig docs
https://learn.microsoft.com/en-us/powershell/module/teams/set-teamsenvironmentconfig?view=teams-ps

> **PRE-REQUISITES steps for** [Office 365 operated by 21Vianet](https://learn.microsoft.com/en-us/microsoft-365/admin/services-in-china/services-in-china?view=o365-21vianet) 
> - Install or upgrade MicrosoftTeams powershell module with version >= 5.2.0

Reviewer 2: ```suggestion
**PRE-REQUISITES steps for** [Office 365 operated by 21Vianet](/microsoft-365/admin/services-in-china/services-in-china?view=o365-21vianet) 
- Install or update the Microsoft Teams PowerShell module to version 5.2.0 or later
```

> Moves one or more user accounts enabled for Skype for Business Server to TeamsOnly (or the reverse). This cmdlet also can be used to move on-premises users from one pool to another. 
> 
> **PRE-REQUISITES steps for** [Office 365 operated by 21Vianet](https://learn.microsoft.com/en-us/microsoft-365/admin/services-in-china/services-in-china?view=o365-21vianet) 
> - Install or upgrade MicrosoftTeams powershell module with version >= 5.2.0

Reviewer 3: ```suggestion
Moves one or more user accounts enabled for Skype for Business Server to Teams Only (or the other way around). This cmdlet can also be used to move on-premises users from one pool to another.

Prerequisite steps for [Office 365 operated by 21Vianet](/microsoft-365/admin/services-in-china/services-in-china):
- Install or update Microsoft Teams PowerShell module to version 5.2.0 or later.
```

> - Run Set-TeamsEnvironmentConfig -TeamsEnvironmentName TeamsChina

Reviewer 3: ```suggestion
- Run Set-TeamsEnvironmentConfig -TeamsEnvironmentName TeamsChina.
```

> Above command needs to be run once per machine(SFB on-prem FE machine, admin toolbox) where move-csuser is triggered in China environment. For more information, refer to [Set-TeamsEnvironmentConfig](https://learn.microsoft.com/en-us/powershell/module/teams/set-teamsenvironmentconfig?view=teams-ps)

Reviewer 2: ```suggestion
For more information, see [Set-TeamsEnvironmentConfig](/powershell/module/teams/set-teamsenvironmentconfig)
```
I would remove this sentence as I think it's confusing, they can go the cmdlet article for more information.

> Above command needs to be run once per machine(SFB on-prem FE machine, admin toolbox) where move-csuser is triggered in China environment. For more information, refer to [Set-TeamsEnvironmentConfig](https://learn.microsoft.com/en-us/powershell/module/teams/set-teamsenvironmentconfig?view=teams-ps)

Reviewer 3: ```suggestion
For more information, see [Set-TeamsEnvironmentConfig](/powershell/module/teams/set-teamsenvironmentconfig).
```

Author: @tseward  Are you suggesting to remove this line "Above command needs to be run once per machine(SFB on-prem FE machine, admin toolbox) where move-csuser is triggered in China environment. " , is yes, why is that? If not, then only suggested change is instead of "refer to " use "see" and shorten the url? Is that the last thing to do here?
